### DISCUSSION TITLE
layer has @types (~3MB)

### DISCUSSION THREAD

**Discussion by Muthuveerappanv** [aws-powertools/powertools-lambda-typescript#1170] [General]
### [Layers](https://awslabs.github.io/aws-lambda-powertools-typescript/latest/#install) when unzipped (6.0M)
The @types directory occupies 3.6M in the layers data, Since @types aren't used at runtime, they can be readily removed 
![Screen Shot 2022-11-16 at 12 25 00 PM](https://user-images.githubusercontent.com/33663725/202302884-5764cdec-3078-4373-9f3a-a92cca611935.png)

#### Without @types (2.4 M)
![Screen Shot 2022-11-16 at 4 59 59 PM](https://user-images.githubusercontent.com/33663725/202303661-7db35edd-bfed-4316-ab1d-0912506b7296.png)

I did `rm -rf node_modules/@types`

### Using tools like [node-prune](https://github.com/tj/node-prune)

I have had much success with using node-prune as well, it takes away unused, files and reduces the size of `node_modules` by a good number, but has to be thoroughly tested. 

![image](https://user-images.githubusercontent.com/33663725/202306077-ed38cb41-2e69-409a-8f9e-fbbbe44128e7.png)

**Comment by dreamorosi** [comment]
Hi @Muthuveerappanv thank you for reporting this, we are going to investigate and try to come up with a solution. Ideally we'd like to shave some data if this doesn't break things. I have opened an issue to track this effort #1171

**Reply by dreamorosi** [reply]
Just to follow up on this, the fix was merged and the types have been removed.
